newlib/libm/ld/common/k_rem_pio2.c

```diff
@@ -118,7 +118,9 @@
  *
  */
 
-#ifdef __GNUC__
```

Comment:
same here; ld/common are files included by ../ equivalents which will have pulled in cdefs.h